src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/RequestExecutionQueue.cs

```diff
@@ -201,7 +204,7 @@ private async Task ProcessQueueAsync()
                         // via NFW, though these errors don't put us into a bad state as far as the rest of the queue goes.
                         // Furthermore we use Task.Run here to protect ourselves against synchronous execution of work
                         // blocking the request queue for longer periods of time (it enforces parallelizabilty).
-                        _ = Task.Run(() => work.StartRequestAsync(cancellationToken), cancellationToken);
+                        _ = Task.Run(() => work.StartRequestAsync(context, cancellationToken), cancellationToken);
```

Comment:
Out of curiosity I was comparing this code to the previous code, and reading the comment, and it seems like we've lost the NFW reporting it talks about:
https://github.com/dotnet/roslyn/blob/release/dev17.4/src/Features/LanguageServer/Protocol/Handler/RequestExecutionQueue.cs#L220

Is that handled elsewhere now? It's a little hard to know by just looking at the file differences, since all of the telemetry and activity ID stuff has also moved.

Reply:
indeed we did.  However this is in the common layer so I'm not exactly certain how to fix this

Reply:
ahhh yes, of course. layering